channels/remdesk/client/remdesk_main.c

```diff
@@ -795,6 +786,10 @@ static VOID VCAPITYPE remdesk_virtual_channel_open_event_ex(LPVOID lpUserParam,
 			break;
 
 		case CHANNEL_EVENT_WRITE_COMPLETE:
+		{
```

Comment:
Should handle `CHANNEL_EVENT_WRITE_CANCELLED` too

Reply:
done